### PR TITLE
Several MTE-2812  [UI Tests Menu/Toolbar] - fixes for auto tests

### PR DESCRIPTION
**Pull request by dragosb01**
## :scroll: Tickets
https://mozilla-hub.atlassian.net/browse/MTE-2812

## :bulb: Description
Several fixes for tests affected by the toolbar redesign

### CODE REVIEW

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift [hunk 1]
```diff
@@ -374,36 +373,39 @@ class SearchTests: BaseTestCase {
...
-        try app.performAccessibilityAudit { issue in
-            guard let element = issue.element else { return false }
+        if !iPad() {
+            try app.performAccessibilityAudit { issue in
+                guard let element = issue.element else { return false }
 
-            var shouldIgnore = false
-            // number of tabs in navigation toolbar
-            let isDynamicTypeTabButton = element.label == "1" &&
+                var shouldIgnore = false
```
Comment (isabelrios) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift:380]: is this the proper indentation?

**Review by isabelrios — APPROVED** [review]
LGTM just nit comment

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift (hunk 1 above)
Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift:380]: i will have a look in the next PR

Comment (dragosb01) [reply firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift:380]: looked over and yes this is the indentation that xcode is doing

**Comment by dragosb01** [comment]
@Mergifyio backport release/v134

File: firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift [hunk 2]
```diff
@@ -270,7 +270,6 @@ class SearchTests: BaseTestCase {
             // Reload icon is displayed.
             mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton])
             XCTAssertEqual(app.buttons[AccessibilityIdentifiers.Toolbar.addNewTabButton].label, "New Tab")
-            navigator.performAction(Action.CloseURLBarOpen)
```
Comment (clarmso) [inline firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift:273]: @mdotb-moz We may need to put this back after we get an answer about the behaviour of the keyboard.
https://github.com/mozilla-mobile/firefox-ios/issues/23436

**Pull request merged**